contributing/ci-omero.txt

```diff
@@ -347,3 +383,16 @@ Jenkins.
            :file:`components/tools/OmeroWeb/target/reports` from
            :term:`OMERO-DEV-breaking-integration`,
         #. Generates pytest report
+
+    :jenkinsjob:`WEB-DEV-breaking-integration`
```

Comment:
This job is not set yet, is the plan to add one (to run breaking robot tests) or should we use WEB-DEV-breaking-deploy for now?